README.md

```diff
@@ -32,18 +32,20 @@ Flags:
   -p, --port string        The port of the HTTP server
 ```
 
-## Eval a policy
+## Evaluate the input by a policy list
```

Comment:
is it possible to make policy-man more generic.. that it simply executes and returns results of all rules. And only in xpanse, we can build a limitation that the rego rules must contain allow/deny rules?

Reply:
If we do it like this, then the response of this API will not be a stable structure. and the Xpanse must deserialize the response and check the allow/deny rules itself.

Reply:
ok. Then we can provide two services. one with specific allow and deny.. and one with a generic one.